apps/web/components/MailingListSignup/MailingListSignup.tsx

```diff
@@ -21,7 +20,7 @@ interface FormProps {
 }
 
 interface MailingListSignupProps {
-  namespace: GetNamespaceQuery['getNamespace']
+  namespace: Record<string, string>
```

Comment:
Hmmm... are the generated types failing?
I'd be interested to know how

Reply:
We are JSON parsing the object before it is passed in as props so the typing is wrong. I could've changed this in another PR but I changed it here.